### PR TITLE
The spot header

### PR DESCRIPTION
**Pull request by carinegordillo**

### CODE REVIEW

File: src/main/webapp/createEvent.html [hunk 1]
```diff
@@ -2,16 +2,23 @@
 <html>
     <head>
         <meta charset="UTF-8">
-        <title>Create Event</title>
+        <title>The Spot - Create Events</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
```
Comment (jmuzy-sps) [inline src/main/webapp/createEvent.html:7]: You have version 3.5.1 below, so idk that you need this one as well

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -19,6 +19,22 @@ body #map{
...
 
+.web-title{
+    font-weight: 900;
+    font-size: 50px;
+    text-align: center;
+    font-family: 'Trebuchet MS', sans-serif;
+    background-color:#353a3f;
+    padding-bottom: 0px;
+    color:#51cf2b;
+    outline:white;
```
Comment (jmuzy-sps) [inline src/main/webapp/style.css]: nit: here and below, alphabetize the attributes

File: src/main/webapp/createEvent.html [hunk 3]
```diff
@@ -2,16 +2,23 @@
...
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
         <link rel="stylesheet" href="style.css">
         <link rel="stylesheet" 
             href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
             integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" 
             crossorigin="anonymous">        
-        <script src="createEvent.js"></script>
+        <script src="script.js"></script>
```
Comment (jmuzy-sps) [inline src/main/webapp/createEvent.html]: Just want to make sure this is intended, I feel like I've seen this script change a few times

**Review by jmuzy-sps — CHANGES_REQUESTED** [review]

File: src/main/webapp/createEvent.html (hunk 3 above)
Comment (carinegordillo) [reply src/main/webapp/createEvent.html]: yes it's intended, I was going to make a js specifically for the CreateEvent.html to try and add some things to the modal but I ended up not doing it

File: src/main/webapp/createEvent.html (hunk 1 above)
Comment (carinegordillo) [reply src/main/webapp/createEvent.html:7]: just tested it out and I needed the 3.4.1, not the 3.5.1

**Review by jmuzy-sps — APPROVED** [review]
Thanks Carine!

**Review by valvarez1218 — APPROVED** [review]
Looks good to me!

**Pull request merged**